src/middleware/externalRequests.js

```diff
@@ -26,7 +26,7 @@ const validateRequestStatusCode = allowedStatuses => {
   }
 }
 
-const performRequest = async (requestDetails, ctx) => {
+const performLookupRequest = async (requestDetails, ctx) => {
```

Comment:
Not sure if this makes a big difference or not, but should we not have the `ctx` as the first variables in the function? Just to try and keep it aligned with the other functions. Unless there is a specific reason for this change?